lib/js/index.js

```diff
@@ -342,7 +349,7 @@ class DateTimePicker extends Events {
   setDate(date) {
     const m = moment(date);
     const month = m.format('MMM');
-    const day = m.format('Do');
+    const day = m.format('D');
```

Comment:
Let me check if there is an option in moment that I can use depending on the language